src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -555,7 +568,9 @@
         this.partners = []
         var recipients
         try {
+          this.isLoading = true
           recipients = await this.$lokapi.searchRecipients(this.searchName)
+          this.isLoading = false
```

Comment:
to move below the catch, to be sure that the loading picture disappear if there is an error ?

Reply:
@stephansainleger  you are right, i corrected it